File: src/main/java/io/github/dylmeadows/eontimer/model/resource/CssResource.java
package io.github.dylmeadows.eontimer.model.resource;

import io.github.dylmeadows.common.core.io.Resource;
import lombok.Getter;
import lombok.RequiredArgsConstructor;

@Getter
@RequiredArgsConstructor
public enum CssResource implements Resource {
    EON_TIMER("io/github/dylmeadows/eontimer/css/EonTimer.css");

    private final String path;
}

File: src/main/kotlin/io/github/dylmeadows/eontimer/model/resource/CssResource.kt
package io.github.dylmeadows.eontimer.model.resource

import io.github.dylmeadows.common.core.io.Resource

enum class CssResource(private val path: String) : Resource {
    MAIN("main.css");

    override fun getPath(): String {
        return "$BASE_RESOURCE_PATH/css/${this.path}"
    }
}